settings.gradle

```diff
@@ -95,6 +95,12 @@ project(':secure-storage-api').projectDir = new File('secure-storage/secure-stor
 project(':secure-storage-impl').projectDir = new File('secure-storage/secure-storage-impl')
 project(':secure-storage-store').projectDir = new File('secure-storage/secure-storage-store')
 
+"autofill".with {
```

Comment:
This different to how we currently add new modules atm so it adds a bit of inconsistency (mainly introducing the modules sub folder). I remember you brought this up before in the Android sync. I think it is worth bringing it up again and maybe suggesting to compile all modules into this modules folder if it's something we want to do going forward.

I don't really have a strong opinion on it and I also am not sure if it has any other implications. I think it just makes it slightly confusing to distinguish submodules vs modules.